Fix globals, part 6: `performance` (#33339)
mdn/content: files/en-us/web/api/window/performance/index.md

---
title: "Window: performance property"
short-title: performance
slug: Web/API/Window/performance
page-type: web-api-instance-property
browser-compat: api.performance
---

{{APIRef("Performance API")}}

> The **`performance`** property of the {{domxref("Window")}} interface returns a {{domxref("Performance")}} object, which can be used to gather performance information about the context it is called in window.

Reviewer: ```suggestion
The **`performance`** property of the {{domxref("Window")}} interface returns a {{domxref("Performance")}} object, which can be used to gather performance information about code running in the window's scope.
```

Performance entries are per context. If you create a mark on the main thread (or other worker), you cannot see it in a worker thread, and vice versa.

## Value

> A {{domxref("Performance")}} object offering access to performance and timing-related information for the context it is called on window.

Reviewer: ```suggestion
A {{domxref("Performance")}} object offering access to performance and timing-related information about code running in the window's scope.
```

> ## Performance API availability

Reviewer: We shouldn't this section here - it's useless for people reading about `Window.performance`. Instead, in `Worker.performance`, we should list the APIs that _are_ supported in workers.
